spec/features/edit_a_document_spec.rb

```diff
@@ -3,62 +3,46 @@
 require "spec_helper"
 
 RSpec.feature "Edit a document" do
-  DocumentTypeSchema.all.each do |schema|
-    next if schema.managed_elsewhere
-
-    scenario "User edits #{schema.name}" do
-      @schema = schema
-
-      given_there_is_a_document
-      when_i_go_to_edit_the_document
-      and_i_fill_in_the_content_fields
-      then_i_see_the_document_is_saved
-      and_the_preview_creation_succeeded
-    end
-
-    def given_there_is_a_document
-      create(:document, document_type: @schema.id)
-    end
-
-    def when_i_go_to_edit_the_document
-      visit document_path(Document.last)
-      click_on "Edit document"
-      @request = stub_publishing_api_put_content(Document.last.content_id, {})
-    end
+  scenario "User edits a document" do
+    given_there_is_a_document
+    when_i_go_to_edit_the_document
+    and_i_fill_in_the_content_fields
+    then_i_see_the_document_is_saved
+    and_the_preview_creation_succeeded
+  end
 
-    def and_i_fill_in_the_content_fields
-      @schema.contents.each do |field|
-        if field.type == "govspeak"
-          fill_in "document[contents][#{field.id}]", with: "Some govspeak text."
-        else
-          raise "You'll have to write some code here to fill in a #{field.type} field"
-        end
-      end
+  def given_there_is_a_document
+    body_field = { id: "body", label: "Body", type: "govspeak", validations: [] }
+    document_type_schema = build(:document_type_schema, contents: [body_field])
+    contents = { body: "Existing body" }
+    create(:document, document_type: document_type_schema.id, contents: contents)
+  end
 
-      fill_in "document[summary]", with: "A summary of the release."
-      click_on "Save"
-    end
+  def when_i_go_to_edit_the_document
+    visit document_path(Document.last)
+    click_on "Edit document"
+    expect(page).to have_content("Existing body")
+  end
 
-    def then_i_see_the_document_is_saved
-      expect(page).to have_content "A summary of the release."
+  def and_i_fill_in_the_content_fields
+    fill_in "document[summary]", with: "A summary of the release."
```

Comment:
The summary is making a sneaky first appearance here. Is it necessary for this test?